doc/adr/0002-use-component-level-micro-frontends.md

```diff
@@ -0,0 +1,29 @@
+# 2. Use component-level micro frontends
+
+Date: 2020-03-25
+
+## Status
+
+Proposed
+
+## Context
+
+Having [separate micro frontends for pages](0001-use-micro-frontends.md) requires duplication of the header and footer.
+
+We expect users will want to change the content of both, as well as their styling and behaviour.
+
+## Decision
+
+We will use [Edge Side Includes (ESIs)](https://www.w3.org/TR/esi-lang), limited to `<esi:include>`.
+
+Each component will have 2 ESIs: 1 for the `<head>` and 1 for the `<body>`.
+
+Each page-level micro frontend will process the ESIs if it is not behind a trusted proxy that can do so. They will
```

Comment:
does it have to be a trusted proxy, or can just be any proxy that adds this header? Thinking about making tests for various scenarios simple.

Reply:
Guess this is detail that is not part of the decision (both being behind a proxy and trusting it, even though both are needed).

Reply:
Move to `Consequences`?